README.md

```diff
@@ -224,26 +224,9 @@ Or in YAML
       encryptor:
         password: ${JASYPT_ENCRYPTOR_PASSWORD:}
 
-
 # Housekeeping Vacuum Tool
-A tool that can be used to interrogate and remove orphaned data which has been scheduled for Housekeeping.
```

Comment:
I've moved this whole section to the VT README so we don't duplicate this and only have to update things in one place.